demos/logging-stack/logging_stack.h

```diff
@@ -41,7 +41,11 @@
 
 /* Common macro for all logging interface macros. */
 /* TODO - Replace printf with an implementation function. */
-#define SdkLog( string )    printf string
+#if !defined( DISABLE_LOGGING )
+    #define SdkLog( string )    printf string
```

Comment:
Where are the `( )`?  This won't compile, is that the intent?

Reply:
I believe this is the reason why all of our logs use double parenthesis `( ( ) )` instead of single `( )`. I think it was made this way since variable arguments in macros are not supported in C89, so all of it is passed at once to printf